Header header
geometry_msgs/Point position
geometry_msgs/Vector3 attitude
geometry_msgs/Twist velocity
geometry_msgs/Accel acceleration
uint32 sequenceNr